authors/apps/authentication/tests/test_login.py

```diff
@@ -0,0 +1,33 @@
+from .test_base import BaseTest
+from rest_framework.views import status
+from .test_data import (valid_user, valid_login,
+                        wrong_password, wrong_email, missing_password_key)
+
+
+class UserLoginTest(BaseTest):
+    def test_valid_login(self):
```

Comment:
@Peace-Apple  and @CryceTruly thank you for the good work, i think it would be better if you guys added more doc strings in your tests, so that you can provide more clarity on your code.

Reply:
okay @BagzieGracious ,  thanks for the feedback, working on it

Reply:
Okay thanks